photo-effects/src/components/Dashboard/DashNav/DashNav.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react'
+import { NavLink } from 'react-router-dom';
+import './dashNav.css';
+
+
+export class DashNav extends Component {
+
+
+
+//push to landing
+
+  render() {
+    return (
+      <nav className="dashNav">
+      <NavLink to="/">
+        <img style={{width:'250px'}} src="https://files.slack.com/files-pri/T4JUEB3ME-FMKCEGRQT/asset_8.png" alt="logo"/>
```

Comment:
change the img sizing to a percentage instead so that it changes size along with the navbar. that way it stays responsive.

Reply:
Okay so change the width: 250px to a percentage. Got it and good insight!